examples/ra-sim/main.py

```diff
@@ -0,0 +1,87 @@
+from argparse import ArgumentParser
+from time import perf_counter
+from typing import Any, Dict, Tuple
+
+import gym
+gym.logger.set_level(40)
+
+import sim
+from utils import ResultsManager, params_str_template, results_str_template
+from reinforced_lib import RLib
+from reinforced_lib.agents import ThompsonSampling
+from reinforced_lib.exts import IEEE_802_11_ax
+
+
+def run(ra_sim_args: Dict[str, Any], seed: int) -> Tuple[str, float, float, float]:
+    """
+    Run example simulation in the Rate Adaptation Simulator.
+
+    Parameters
+    ----------
+    ra_sim_args : dict
+        Arguments passed to the simulator.
+    seed : int
+         An integer used as the random key.
+
+    Returns
+    -------
+    results : tuple[str, float, float, float]
+        Tuple containing all results in CSV format, mean MCS, mean data rate and estimated mean throughput.
+    """
+
+    rl = RLib(
+        agent_type=ThompsonSampling,
+        ext_type=IEEE_802_11_ax
+    )
+    rl.init(seed)
+
+    env = gym.make('RASimEnv-v1')
+    state, _ = env.reset(seed=seed, options=ra_sim_args)
```

Comment:
Tylko takie info, w starszej wersji gym'a wywalało mi się w tym miejscu (krzyczał że "The obs returned by the 'reset()' method must be a dict, actual type: «class 'tuple'>"), ale jak zaktualizowałem do wersji z requirements to elegancko się uruchomiło :D

Reply:
U mnie to samo, ale stwierdziłem, że lepiej będzie przejść na nową wersję gym'a i już zrobić to wg nowego API.